data_utils2.py

```diff
@@ -0,0 +1,281 @@
+import json
+import random
+import os
+from typing import Iterator, Optional, Dict, Iterator, List, Optional, Sequence, Union, Callable
```

Comment:
The file names `data_utils2` and `data_utils` are confusing. What is the difference?